webapp/app/partials/main/thResultCounts.html

```diff
@@ -1,10 +1,5 @@
-    <span class="btn btn-default btn-sm btn-resultset result-status-total-count"
-          tabindex="0" role="button"
-          ng-click="toggleJobs()"
-          title="total job count - toggle jobs">
-            <i class="fa fa-list fa-lg result-set-fa-icon"></i>
-            <span class="result-status-total-value"></span>
-    </span>
-    <span class="result-status-count-group">
-        <th-result-status-count ng-repeat="resultStatus in statusList"/>
-    </span>
+<span class="result-set-progress">
+    <span ng-if="percentComplete == 100">- Complete -</span>
+    <span ng-if="percentComplete < 100"
+          title="% of jobs completed vs. pending and running">{{percentComplete}}% - {{inProgress}} in progress</span>
```

Comment:
The percentage shown is "completed / total". The wording in the title to me reads as "pending+running / completed" or something along those lines.

For the title, how about this: "Proportion of jobs that are complete" :-)